cerebus/cbsdk_cython.pxd

```diff
@@ -3,13 +3,234 @@
 
 @author: dashesy
 
-Purpose: C interface for cbpy wrapper 
+Purpose: Cython interface for cbsdk_small
```

Comment:
Should be cbsdk_helper, not cbsdk_small. (The latter was a name I tried on but decided I didn't like.)